duplicate.js

```diff
@@ -0,0 +1,34 @@
+function duplicate(arr){
+  var a =new Set(arr);
+  if (arr.length === a.size){
+     return false;
+  }
+  else {
+      return true;
+  }
+}
+
+var unitTest={};
+
+unitTest.testDupl = function(method){
```

Comment:
This is not the way unit tests are usually written in javascript, please search for a common way and adjust your implementation.